assets/src/design-system/components/button/constants.js

```diff
@@ -32,3 +32,5 @@ export const BUTTON_VARIANTS = {
   SQUARE: 'square',
   ICON: 'icon',
 };
+
+export const BUTTON_TRANSITION_TIMING = '0.6s ease 0s';
```

Comment:
wanted to share this with the context menu so that the buttons animate at the same speed even though the styling is different